Makefile

```diff
@@ -304,7 +311,7 @@ endif
 .PHONY: clean tools tidy clean-tools $(TOOLDIRS) libagbsyscall
 
 # Ensure required directories exist
-$(shell mkdir -p $(C_BUILDDIR) $(DATA_ASM_BUILDDIR) $(SOUND_ASM_BUILDDIR) $(SONG_BUILDDIR) $(MID_BUILDDIR))
+$(shell mkdir -p $(C_BUILDDIR) $(ASM_BUILDDIR) $(DATA_ASM_BUILDDIR) $(SOUND_ASM_BUILDDIR) $(SONG_BUILDDIR) $(MID_BUILDDIR))
```

Comment:
Why have you added this back in? I removed because it's not need in this repo? :/

Reply:
For consistency between both repos! :P
It's much easier to copy-paste changes between both of them instead of having to painstakingly go through the maze that is this Makefile to find out what to change where specifically in one of the two targets.

Reply:
Ok, fair enough, but I would imagine we are not going to merge the repos together until we have the asm removed from both?

Reply:
Or is this just so that you can copy and paste any change from here across?

Reply:
Mostly the latter, but yeah, it'll be some time before we can think about merging them.